cpp/include/cudf/detail/get_value.cuh

```diff
@@ -48,11 +49,8 @@ T get_value(column_view const& col_view, size_type element_index, rmm::cuda_stre
   CUDF_EXPECTS(data_type(type_to_id<T>()) == col_view.type(), "get_value data type mismatch");
   CUDF_EXPECTS(element_index >= 0 && element_index < col_view.size(),
                "invalid element_index value");
-  T result;
-  CUDF_CUDA_TRY(cudaMemcpyAsync(
-    &result, col_view.data<T>() + element_index, sizeof(T), cudaMemcpyDefault, stream.value()));
-  stream.synchronize();
```

Comment:
This code is removed in https://github.com/rapidsai/cudf/pull/17520

Reply:
This comment somehow moved to a different file :D
Should be resolved now, updated the branch and resolved conflicts.